react-time-track/src/services/user.js

```diff
@@ -0,0 +1,25 @@
+const API_USERS = "http://localhost:3000/api/users";
+
+async function userList() {
+  const userList = await fetch(API_USERS, {
+    method: "GET",
+    credentials: "include",
+    headers: {
+      "Content-Type": "application/json"
+    }
+  }).then(response => response.json());
+  return userList;
+}
+
+async function roleUsers() {
+  const role = await fetch(API_USERS, {
+    method: "GET",
+    credentials: "include",
+    headers: {
+      "Content-Type": "application/json"
+    }
+  }).then(response => response.json());
+  return role;
```

Comment:
Hi, I seem to me that you are duplicating this function with the before one, because it does the same. What do you think about deleting it?

Reply:
You're right! I forgot to delete it. I am going to delete now.